### DISCUSSION TITLE
Environment value for `tart push ... --populate-cache`

### DISCUSSION THREAD

**Discussion by olejnjak** [cirruslabs/tart#736] [Ideas]
Hi guys,

in my workflow I very often forget to use the `--populate-cache` flag when pushing a VM. I usually push a "VM-chain" - similarly to your macos templates (vanilla -> base -> xcode -> ...) and forgetting it is pretty frustrating.

I thinkg that implementing an environment variable, that would allow changing the default would be pretty elegant solution how to stick with current behavior but helping users not to forget that.

Are you open to such simple PR? My suggestion would be `TART_PULL_POPULATE_CACHE=1`.

Thanks in advance.

**Comment by fkorotkov** [comment]
I wish it was supported by the upstream via https://github.com/apple/swift-argument-parser/issues/4. There was some work done but it staled. I've [poked the thread](https://github.com/apple/swift-argument-parser/issues/4#issuecomment-1954330753) to see if there is an interest to add such support upstream.
